tools/seed/seed-users-and-fleets.sh

```
@@ -0,0 +1,326 @@
+#!/bin/bash
+
+# Seed users, fleets, policies, and reports for local development.
+# See tools/seed/README.md for full documentation.
+
+set -euo pipefail
+
+source "$FLEET_ENV_PATH"
```

Shellcheck (0.11.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)